src/GeneratedInjectorDelegator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Di;
+
+use Interop\Container\ContainerInterface;
+use Zend\ServiceManager\Factory\DelegatorFactoryInterface;
+
+class GeneratedInjectorDelegator implements DelegatorFactoryInterface
```

Comment:
Tend service manager is no dependency of zend-di, therefore factories should not rely on its interfaces to be present. This should be a generic invokable accepted by zend service manager.

```suggestion
class GeneratedInjectorDelegator
```